packages/common/src/utils/urlFormatting.ts

```diff
@@ -103,8 +103,8 @@ export const mediaURL = ({
   return createPath(PATH_MEDIA, { id: media.mediaid, title: slugify(media.title) }, { r: playlistId, play: play ? '1' : null, e: episodeId });
 };
 
-export const playlistURL = (id: string, title?: string) => {
-  return createPath(PATH_PLAYLIST, { id, title: title ? slugify(title) : undefined });
+export const playlistURL = ({ id, title, type }: { id: string; title?: string; type?: 'playlist' | 'content_list' }) => {
+  return createPath(PATH_PLAYLIST, { id, title: title ? slugify(title) : undefined }, { t: type });
```

Comment:
I would keep this as succinct as possible and try to avoid changing the existing playlist routing:
```suggestion
  return createPath(PATH_PLAYLIST, { id, title: title ? slugify(title) : undefined }, type === 'content_list' ? { t: 'c' } : {});
```